AzureStack/2015-04-01/swagger/GalleryAdmin.json

```diff
@@ -0,0 +1,502 @@
+            "in": "path",
+            "description": "Identity of the gallery item. Includes publisher name, item name and may include version separated by period character.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registrationResourceDefinition",
+            "in": "body",
+            "description": "Resource definition of the gallery item.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ResourceDefinition"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ProviderGalleryRegistrations"
+        ],
+        "description": "Deletes a specific gallery registration",
+        "operationId": "ProviderGalleryRegistrations.Delete",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "galleryItemName",
+            "in": "path",
+            "description": "Identity of the gallery item. Includes publisher name, item name and may include version separated by period character.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.gallery.providers/galleryItems": {
+      "get": {
+        "tags": [
+          "ProviderGalleryRegistrations"
+        ],
+        "description": "Lists gallery registrations",
+        "operationId": "ProviderGalleryRegistrations_List",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
```

Comment:
This is an anti-pattern from the ARM guidelines. The response object should have "value" as a top level property which is of type array. The response object may or may not have a nextLink.